dm/syncer/data_validator.go

```diff
@@ -198,3 +386,158 @@ func (v *DataValidator) Stage() pb.Stage {
 	defer v.RUnlock()
 	return v.stage
 }
+
+func (v *DataValidator) startValidateWorkers() {
+}
+
+func (v *DataValidator) dispatchRowChange(key string, row *rowChange) {
+}
+
+func (v *DataValidator) processRowsEvent(header *replication.EventHeader, ev *replication.RowsEvent) error {
+	sourceTable := &filter.Table{
+		Schema: string(ev.Table.Schema),
+		Name:   string(ev.Table.Table),
+	}
+	fullTableName := sourceTable.String()
+	if _, ok := v.unsupportedTable[fullTableName]; ok {
+		return nil
+	}
+
+	targetTable := v.syncer.route(sourceTable)
+	tableInfo, err := v.syncer.schemaTracker.GetTableInfo(sourceTable)
+	if err != nil {
+		if schema.IsTableNotExists(err) {
```

Comment:
can we use `sync.skipRowsEvent` like syncer to avoid this error happen ?  

> i guess `skipRowsEvent` have better perfoemance than `GetTableInfo` seems schema need use disk io

Reply:
added before `GetTableInfo`